styles/keybr.com/catppuccin.user.css

```diff
@@ -0,0 +1,130 @@
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    /* Start styling your website here: */
+    --primary-d2: @overlay0;
+    --primary-d1: @overlay0;
+    --primary: @base;
+    --primary-l1: @crust;
+    --primary-l2: @mantle;
+    --secondary-d1: @subtext0;
+    --secondary: @text;
+    --secondary-l1: @subtext0;
+    --secondary-l2: @accent-color;
+    --secondary-f1: @subtext0;
+    --secondary-f2: @subtext0;
+    --accent-d2: @accent-color;
+    --accent-d1: @accent-color;
+    --accent: @accent-color;
+    --accent-l1: @accent-color;
+    --accent-l2: @accent-color;
+    --error-d1: @red;
+    --error: @red;
+    --error-l1: @red;
+    --shadow-color: rgba(red(@crust), green(@crust), blue(@crust), 88);
+    --slow-key-color: @red;
+    --fast-key-color: @green;
+    --Chart-speed__color: @green;
+    --Chart-accuracy__color: @peach;
+    --Chart-complexity__color: @pink;
+    --Chart-threshold__color: @maroon;
+    --Chart-hist-h__color: @lavender;
+    --Chart-hist-m__color: @maroon;
+    --Chart-hist-r__color: @mauve;
+    --KeyboardKey-pointer__color: @accent-color;
+    --KeyboardKey-symbol__color: @crust;
+    --pinky-zone-color: @green;
+    --ring-zone-color: @teal;
+    --middle-zone-color: @yellow;
+    --left-index-zone-color: @blue;
+    --right-index-zone-color: @maroon;
+    --thumb-zone-color: @mauve;
+    --effort-0-color: @crust;
+    --effort-1-color: @crust;
+    --effort-2-color: @accent-color;
+    --effort-3-color: @accent-color;
+    --Value--more__color: @green;
+    --Value--less__color: @red;
+    --KeyboardKey-button--depressed__color: @accent-color;
+    --LessonKey--included__color: @crust;
+    --LessonKey--excluded__color: @crust;
+    --DailyGoal-bar__color: @accent-color;
+    .nc1oZcWRbC {
```

Comment:
You should leave a comment here since this an auto-generated class - or even better try to find a more resilient selector. They do have a theme import/export feature using JSON files with variable names in them, so if anything isn't themed by these variables it's probably a bug and could be fixed at https://github.com/aradzie/keybr.com.

Reply:
It's the current letter. As the current letter is not recolored in the original theme there is not a variable to modify it's color.

Reply:
Probably no need to recolor it here then?